nomic/embed.py

```diff
@@ -132,17 +154,17 @@ def run_inference(batch):
 
     def resize_pil(img):
         width, height = img.size
-        #if image is too large, downsample before sending over the wire
+        # if image is too large, downsample before sending over the wire
         max_width = 512
         max_height = 512
         if max_width > 512 or max_height > 512:
-            downsize_factor = max(width/max_width, height/max_height)
-            img.resize((width/downsize_factor, height/downsize_factor))
+            downsize_factor = max(width / max_width, height / max_height)
+            img.resize((width / downsize_factor, height / downsize_factor))
         return img
 
     def send_request(i):
         image_batch = []
-        shard = images[i:i+IMAGE_EMBEDDING_BATCH_SIZE]
+        shard = images[i : i + IMAGE_EMBEDDING_BATCH_SIZE]
```

Comment:
I'm not personally a fan of spaces around the `:` in a slice.

Reply:
yeah agreed, maybe disable this.

Reply:
black is doing this, and it's not configurable.